allennlp/data/vocabulary.py

```diff
@@ -325,11 +365,15 @@ def from_params(cls, params: Params, dataset=None):
         min_count = params.pop("min_count", 1)
         max_vocab_size = params.pop("max_vocab_size", None)
         non_padded_namespaces = params.pop("non_padded_namespaces", DEFAULT_NON_PADDED_NAMESPACES)
+        pretrained_files = params.pop("pretrained_files", {})
+        pretrained_exclusive = params.pop("pretrained_exlusive", False)
         params.assert_empty("Vocabulary - from dataset")
         return Vocabulary.from_dataset(dataset,
                                        min_count,
                                        max_vocab_size,
```

Comment:
Can you add keyword args to these, too?  We've tried to do that with all of our `from_params` methods.  It just helps future-proof the code, in case arguments or argument orders change later.

Reply:
Not sure what you mean. Do you mean add keyword args to the from_params method, or something else?

Reply:
Sorry for not being more clear; I just meant adding `dataset=dataset`, `min_count=min_count` to these lines.